clients/cobol-lsp-vscode-extension/src/__tests__/commands/RenumCommand.spec.ts

```diff
@@ -0,0 +1,216 @@
+    unNumberLines(mockNumberedDocument, editMock, RENUM_RIGHT);
+    expect(replaceMock).toHaveBeenCalledWith(
+      { end: { character: 80, line: 1 }, start: { character: 72, line: 1 } },
+      " ".repeat(8),
+    );
+  });
+  it("no changes if document consists more than 999999 lines", () => {
+    const bigMock = {
+      lineCount: 1000000,
+      lineAt: (_num: number) => ({ text: "" }),
+    };
+    renumberLines(bigMock, editMock, RENUM_LEFT);
+    expect(replaceMock).toHaveBeenCalledTimes(0);
+    expect(vscode.window.showErrorMessage).toHaveBeenCalledWith(
+      "Renumber sequential numbers is not possible above 999999 lines",
+    );
+  });
+  it("check getSequentialNumber against pad 4", () => {
+    expect(getSequentialNumber(0, 6, 4)).toEqual("000100");
+  });
+  it("check getSequentialNumber against line & pad 5", () => {
+    expect(getSequentialNumber(4, 8, 5)).toEqual("00005000");
+  });
+  it("check getSequentialNumber against line 1250 & digit 8", () => {
+    expect(getSequentialNumber(1249, 8, 5)).toEqual("01250000");
+  });
+  it("cheks getSequentialNumber returns properiate value when multiplication exceeds line number", () => {
+    expect(getSequentialNumber(999998, 6, 0)).toEqual("999999");
+  });
+  it("check getSequentialNumber against digit 6 & totalLine is more than 9999", () => {
+    expect(getSequentialNumber(0, 6, 5)).toEqual("000010");
+  });
+  it(
+    "check unNumberLines against digit 6 removes numbering at all lines in 1-6 columns & " +
+      "lines commented out doesn't modified",
+    () => {
+      unNumberLines(mockDocument, editMock, RENUM_LEFT);
+      expect(replaceMock).toHaveBeenCalledTimes(4);
+      expect(replaceMock).toHaveBeenNthCalledWith(
+        1,
+        { end: { character: 6, line: 0 }, start: { character: 0, line: 0 } },
+        " ".repeat(6),
+      );
+      expect(replaceMock).toHaveBeenNthCalledWith(
+        2,
+        { end: { character: 6, line: 1 }, start: { character: 0, line: 1 } },
+        " ".repeat(6),
+      );
+      expect(replaceMock).toHaveBeenNthCalledWith(
+        3,
+        { end: { character: 6, line: 2 }, start: { character: 0, line: 2 } },
+        " ".repeat(6),
+      );
+      expect(replaceMock).toHaveBeenNthCalledWith(
+        4,
+        { end: { character: 6, line: 3 }, start: { character: 0, line: 3 } },
+        " ".repeat(6),
+      );
+    },
+  );
+  it("check unNumberLines against digit 8 removes numbering at all lines in 73-80 columns & ", () => {
+    unNumberLines(mockNumberedDocument, editMock, RENUM_RIGHT);
+    expect(replaceMock).toHaveBeenCalledTimes(2);
+    expect(replaceMock).toHaveBeenNthCalledWith(
+      1,
+      { end: { character: 80, line: 0 }, start: { character: 72, line: 0 } },
+      " ".repeat(8),
+    );
+    expect(replaceMock).toHaveBeenNthCalledWith(
+      2,
+      { end: { character: 80, line: 1 }, start: { character: 72, line: 1 } },
+      " ".repeat(8),
+    );
+  });
```

Comment:
These tests look like extended versions of `"Unnumber Lines removes sequential numbers at 0 to 6 colums"` and `"Unnumber Lines removes sequential numbers at 72 to 80 colums"` and comment status of the line is no longer a factor. So I think these tests can be merged.